tollway/vehicle.py

```diff
@@ -41,7 +41,11 @@ def get_tollways(html: Optional[requests.Response] = None) -> dict:
         for sibling in h2.find_next_siblings():
             if sibling.name == "table":
                 tolls.extend(
-                    [unicodedata.normalize("NFKD", td.text.strip()) for tr in sibling.find_all("tr") if (td := tr.find("td")) is not None]
+                    [
+                        unicodedata.normalize("NFKD", td.text.strip())
+                        for tr in sibling.find_all("tr")
+                        if (td := tr.find("td")) is not None
```

Comment:
walrus! 
still torn haha

Reply:
haha. I like it! It's quite concise and I like the name of it too, ha.